packages/forms-web-app/src/pages/projects/section-51/index/_utils/advice-helpers.js

```diff
@@ -11,6 +11,8 @@ const getAdviceName = ({ organisation, firstName, lastName }, i18n) => {
 	let name = i18n.t('section51.anonymous');
 	if (organisation?.trim()) name = organisation.trim();
```

Comment:
Not critical but would something like this be consistent with others?

const getAdviceName = ({ organisation, firstName, lastName }, i18n) => {
	return  organisation?.trim() ||
		(firstName && lastName ? `${firstName} ${lastName}`
		: firstName || lastName || i18n.t('section51.anonymous'));
};